emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java

```diff
@@ -137,6 +145,84 @@ public EClassifier getCorrespondingEClass(Classifier type) {
     }
   }
 
+  public Enumeration getCorrespondingEnumeration(EEnum eEnum) {
+    if (!eEnumsToEnumerations.containsKey(eEnum)) {
+      if (ePackagesToLanguages.containsKey(eEnum.getEPackage())) {
+        throw new IllegalStateException(
+                "Cannot find corresponding Enumeration for EEnum "
+                        + eEnum.getName()
+                        + " in EPackage "
+                        + eEnum.getEPackage().getName());
+      }
+      processEPackage(eEnum.getEPackage());
+    }
+    if (!eEnumsToEnumerations.containsKey(eEnum)) {
+      throw new IllegalStateException(
+              "Cannot find corresponding Interface for EClass " + eEnum.getName());
```

Comment:
This error message seems incorrect, since we're dealing with enumerations. (The implementation of `getCorrespondingEEnum` has a correct one.)